app/uk/gov/hmrc/eoricommoncomponent/frontend/views/match_name_id_organisation.scala.html

```diff
@@ -34,8 +34,12 @@
 }
 
 @hintText = @{
-    if(organisationType==PartnershipDM) messages(s"cds.matching.name-id-organisation.registered-partnership.name.hint")
-    else messages(s"cds.matching.name-id-organisation.registered-company.name.hint")
+  organisationType match {
+      case PartnershipDM => messages(s"cds.matching.name-id-organisation.registered-partnership.name.hint")
+      case CompanyDM | PartnershipLLpDM => messages(s"cds.matching.name-id-organisation.registered-company.name.hint")
+      case _ => ""
+  }
```

Comment:
We don't want to have empty hint text, so if we don't want a hint in the other scenarios then this should return a None rather than an empty string